config/default/jenkins-release.yaml

```diff
@@ -15,6 +15,9 @@ jenkins:
   master:
     image: jenkins/jenkins@sha256
     imageTag: dfe766e539dc2f47600229c7ed5bb7950b34d2c61f1ae240ef9393cbc0df6e68
+    sidecars:
+      configAutoReload:
```

Comment:
it'll still reload on any changes that cause a reboot

Reply:
I know :p

Reply:
I am working on it, so it's fine as long as we don't reboot the service